app/views/shared/_missing_poster.erb

```diff
@@ -0,0 +1,3 @@
+<div class="missing-poster">
+	<p><%= title %></p>
```

Comment:
This partial used to take a `movie` object and call `movie.title`. Now it just takes the text we want to render (`title`). This means the partial can be used for both movies (that have a `title` attribute) and TV shows (that have a `show_name` attribute).

Reply:
❤️